### PR TITLE
bpo-32133: Improve numbers docs

### PR DESCRIPTION
**Pull request by miguendes**
https://bugs.python.org/issue32133

### CODE REVIEW

**Comment by skirpichev** [comment]
@miguendes, please include issue number in the PR title.

File: Doc/library/numbers.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 The :mod:`numbers` module (:pep:`3141`) defines a hierarchy of numeric
 :term:`abstract base classes <abstract base class>` which progressively define
-more operations.  None of the types defined in this module can be instantiated.
+more operations. None of the types defined in this module are intended to be instantiated.
```
Comment (Fidget-Spinner) [inline Doc/library/numbers.rst]: This isn't anything major, but you don't have to remove/reformat that whitespace. Officially, PEP 12 recommends 2 spaces after every sentence https://www.python.org/dev/peps/pep-0012/#general.

A lot of places in the docs don't follow this though. So I don't recommend trying to re-format other parts of the docs.
```suggestion
more operations.  None of the types defined in this module are intended to be instantiated.
```

**Review by Fidget-Spinner — APPROVED** [review]
Thanks for the patch. LGTM.

File: Doc/library/numbers.rst (hunk 1 above)
Comment (miguendes) [reply Doc/library/numbers.rst]: It wasn't intentional. My IDE did that on save. Will revert that.

Comment (Fidget-Spinner) [reply Doc/library/numbers.rst]: Yep no worries :). My IDE does that on C code too.

**Review by rhettinger — APPROVED** [review]

**Comment by Fidget-Spinner** [comment]
@rhettinger thanks for taking the time to review and merge this. I've added a `needs backport to 3.10` label as well as the current main branch is for 3.11. I hope you don't mind.

**Pull request merged**